client/document/desktop.jsx

```diff
@@ -60,6 +65,7 @@ class Desktop extends React.Component {
 						data-webpack={ true }
 						href={ `/calypso/fallback/build.${ isRTL ? 'rtl.css' : 'css' }` }
 					/>
```

Comment:
The `build.css` chunk is part of the `entrypoint` array: this explicit `link` tag can be removed.